src/components/CallToAction/EmailCaptureForm.js

```diff
@@ -25,12 +25,26 @@ const EmailCaptureForm = ({
   className,
   honeypotValue,
   setHoneypotText,
+  showRecaptcha,
+  setShowRecaptcha,
+  color = 'primary',
 }) => {
   let disabled = submitting || !email || email === '';
   if (newsletterRecaptchaEnabled()) {
     disabled = disabled || !recaptchaResponse || recaptchaResponse === '' || recaptchaExpired;
   }
 
+  const handleEmailChange = (value) => {
+    setEmail(value);
+    if (newsletterRecaptchaEnabled()) {
+      if (value && value.length > 0) {
+        setShowRecaptcha(true);
+      } else {
+        setShowRecaptcha(false);
+      }
+    }
+  };
```

Comment:
Recaptcha's now slide out of the form when needed, rather than looking ugly on the page.